docs/source/changelog.rst

```diff
@@ -1,7 +1,18 @@
 =========
 Changelog
 =========
+.. Future Release
```

Comment:
this should be uncommented since the release already happened

Reply:
Done